community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -152,7 +152,7 @@
     @Description( "Internal log contexts that should output debug level logging" )
     @Internal
     public static final Setting<List<String>> store_internal_debug_contexts = setting( "store.internal_log.debug_contexts",
-            list( ",", STRING ), "" );
+            list( ",", STRING ), "org.neo4j.diagnostics,org.neo4j.cluster.protocol,org.neo4j.kernel.ha" );
```

Comment:
I agree, at least org.neo4j.diagnostics should be INFO then... perhaps ha/cluster as well, but not sure.